src/main/lrsql/admin/routes.clj

```diff
@@ -97,4 +101,4 @@
                 (admin-account-routes common-interceptors secret exp leeway)
                 (admin-cred-routes common-interceptors secret leeway)
                 (when enable-admin-ui
-                  admin-ui-routes))))
+                  (admin-ui-routes common-interceptors)))))
```

Comment:
Realized I missed this - looks like the "convenience" admin UI routes get disabled but the original `/admin/index.html` one doesn't. Something to consider when updating the docs.

Reply:
Wait forgot that `enable-admin-ui` is a totally different var please disregard.